File: algorithm/src/main/java/baekjoon/p02720/Main.java
package baekjoon.p02720;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            for (int solve : solve(sc.nextInt())) {
                System.out.print(solve + " ");
            }

            System.out.println();
        }
    }

    private static int[] solve(int input) {
        int n = input;
        int[] sets = {25, 10, 5, 1};
        int[] ret = new int[sets.length];

        for (int i = 0; i < ret.length; i++) {
            ret[i] = (n / sets[i]);
            n %= sets[i];
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p02720/Main.kt
package baekjoon.p02720

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        println(solve(readLine()!!.toInt()).joinToString(" "))
    }
}

private fun solve(input: Int): Array<Int> {
    var n = input
    val sets = arrayOf(25, 10, 5, 1)
    val ret = Array(sets.size) { 0 }

    for (i in 0 until ret.size) {
        ret[i] = (n / sets[i])
        n %= sets[i]
    }

    return ret
}